win32ss/user/winsrv/usersrv/harderror.c

```diff
@@ -533,6 +534,12 @@ UserpMessageBox(
         case OptionShutdownSystem:
             Type = MB_RETRYCANCEL; // FIXME???
             break;
+        case OptionOkNoWait:
+            /*
+             * At that point showing the balloon failed. Is that correct?
+             */
+            Type = MB_YESNO; // FIXME!
```

Comment:
Not it is not, it's just MB_OK (just check on windows: kill explorer so no shell, then run your test program that sends these hard errors).
Also, OptionOkNoWait should read: "Option OK, no-wait".

Reply:
This was just a revert of my previews changes :)